specification/help/resource-manager/Microsoft.Help/preview/2024-03-01-preview/help.json

```diff
@@ -858,7 +858,7 @@
           "discoverSolutions"
         ],
         "description": "Search for relevant Azure Diagnostics, Solutions and Troubleshooters using a natural language issue summary and subscription.",
-        "operationId": "DiscoverySolutionNLPSubscriptionScope_Post",
+        "operationId": "DiscoverySolutionNLPSubscriptionScope_DiscoverSolutions",
```

Comment:
The 2 DiscoverSolutions may better be in same operation group, and diff by name, e.g. `DiscoverSolutions` and `DiscoverSolutionsBySubscription`

Reply:
Added this suggesstion